help/actions/_posts/2015-02-09-export.md

```diff
@@ -169,8 +176,9 @@ Specifier | Field  | Comments
 **P** | Last Page | prints out the last page number of the article if available 
 **p** | First Page | prints out the first page number of the article if available 
 **pp** | Page Range | prints out the page range numbers of the article if available 
+**pc** | Page Count | prints out the number pages of the article if available
```

Comment:
small typo: "prints out the number **of** pages..."

Reply:
thank you so much.